### DISCUSSION TITLE
无法调整Notification的默认位置？距离上下左右容器的距离

### DISCUSSION THREAD

**Discussion by LiuChao19940201** [irihitech/Semi.Avalonia#413] [Q&A]
是我没找到使用方法，还是目前无法设置呢？
![image](https://github.com/user-attachments/assets/e2216f1f-7bbe-499b-b5fd-b95f660d122f)
![image](https://github.com/user-attachments/assets/03573a17-6fa9-4291-919a-5776d821da2f)

**Comment by rabbitism** [comment]
可以直接在App.xaml中加入设置
```
        <Style Selector="WindowNotificationManager">
            <Setter Property="Margin" Value="50"></Setter>
        </Style>
```
